tests/functionnal/CsvResponse.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2020 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+namespace tests\units\Glpi\Csv;
+
+use Computer;
+use DbTestCase;
+use Glpi\Csv\CsvResponse as Core_CsvResponse;
+use Glpi\Csv\LogCsvExport as CsvLogCsvExport;
+use League\Csv\Reader;
+
+class CsvResponse extends DbTestCase {
+
+   public function testCsvResponse() {
+      ob_start();
+
+      // Create a dummy computer
+      $computer = new Computer();
+      $id = $computer->add([
+         'name'        => 'testExportToCsv 1',
+         'entities_id' => getItemByTypeName('Entity', '_test_root_entity', true),
+      ]);
+
+      $this->integer($id)->isGreaterThan(0);
+      $this->boolean($computer->getFromDB($id))->isTrue();
+
+      // Output CSV
+      Core_CsvResponse::output(new CsvLogCsvExport($computer, []));
+
```

Comment:
I would prefer that ob* methods encapsulate the minimum PHP code to be sue that any failure on test have a minimal impact. Indeed, I am not sure that atoum correctly close an output buffer that would be opened in a test and not closed.
```suggestion
      // Output CSV
      ob_start();
      Core_CsvResponse::output(new CsvLogCsvExport($computer, []));
      $csv_response = ob_get_clean();
```